compiler/one-cmds/one-optimize

```diff
@@ -56,6 +56,12 @@ def _get_parser():
         help='fuse BatchNorm op to Transposed Convolution op')
     circle2circle_group.add_argument(
         '--fuse_bcq', action='store_true', help='apply Binary Coded Quantization')
+    circle2circle_group.add_argument(
+        '--fuse_preactivation_batchnorm', action='store_true', help='fuse BatchNorm operators of pre-activations to Convolution operator')
```

Comment:
can you please split the lines? like,
```
'--fuse_preactivation_batchnorm', 
action='store_true', 
help='fuse BatchNorm operators of pre-activations to Convolution operator')
```
if possible, let's put  help in 90 columns.

Reply:
I've split the lines.